test/e2e/pager.test.js

```diff
@@ -8,83 +8,93 @@ describe('vl-pager', async () => {
         await pager.goToPage('7');
-        const currentPage = await pager.getCurrentPageNumber();
-        assert.isTrue(currentPage == 7);
+        await assert.eventually.equal(pager.getCurrentPage(), 7);
         await pager.reset();
     });
 
-    it('ik kan naar de laatste pagina navigeren', async () => {
+    it('Als gebruiker kan ik naar de laatste pagina en naar de eerste pagina navigeren', async () => {
         const pager = await vlPagerPage.getDefaultPager();
         await pager.goToLastPage();
-        const currentPage = await pager.getCurrentPageNumber();
-        assert.isTrue(currentPage == 10);
+        await assert.eventually.equal(pager.getCurrentPage(), 10);
+        await pager.goToFirstPage();
+        await assert.eventually.equal(pager.getCurrentPage(), 1);
         await pager.reset();
     });
 
-    it('ik kan naar de eerste pagina navigeren', async () => {
+    it('Als gebruiker zie ik kan het totaal aantal resultaten', async () => {
         const pager = await vlPagerPage.getDefaultPager();
-        await pager.goToLastPage();
-        const currentPage = await pager.getCurrentPageNumber();
-        assert.isTrue(currentPage == 10);
-        await pager.goToFirstPage();
-        const firstPage = await pager.getCurrentPageNumber();
-        assert.isTrue(firstPage == 1);
+        await assert.eventually.equal(pager.getTotalItems(), 100);
     });
 
-    it('ik kan het totale aantal resultaten opvragen', async () => {
+    it('Als gebruiker zie ik hoeveel items zichtbaar zijn', async () => {
         const pager = await vlPagerPage.getDefaultPager();
-        const totalResults = await pager.getTotalResults();
-        assert.isTrue(totalResults == 100);
+        await assert.eventually.equal(pager.getItemsPerpage(), 10);
     });
 
-    it('ik kan het totale aantal zichtbare resultaten opvragen', async () => {
+    it ('Als gebruiker zie ik dat een default pager left gealigneerd is', async() => {
         const pager = await vlPagerPage.getDefaultPager();
-        const totalVisibileResults = await pager.getTotalOfDisplayedResults();
-        assert.isTrue(totalVisibileResults == 10);
+        await assert.eventually.isTrue(pager.isAlignedLeft());
+    });
+
+    it('Als gebruiker zie ik een pager gecentreerd staat', async() => {
+        const pager = await vlPagerPage.getPagerCenter();
+        await assert.eventually.isTrue(pager.isAlignedCenter());
     });
 
+    it('Als gebruiker zie ik een pager rechts gealigneerd staat', async() => {
+        const pager = await vlPagerPage.getPagerRight();
+        await assert.eventually.isTrue(pager.isAlignedRight());
+    });
+
+    it('Als gebruiker kan ik op volgende en vorige pagina klikken als de paginatie gedisabled is', async() => {
+        const pager = await vlPagerPage.getPagerZonderPaginaItems();
+        await assert.eventually.isTrue(pager.isPaginationDisabled());
+
+        await assert.eventually.equal(pager.getRange(), "1-10");
```

Comment:
Ik zou getRange() de getallen laten parsen en in een mooi object teruggeven (of getRangeMin en getRangeMax introduceren).